pkgs/applications/version-management/gitlab/default.nix

```diff
@@ -118,7 +118,14 @@ stdenv.mkDerivation {
     rubyEnv rubyEnv.wrappedRuby rubyEnv.bundler tzdata git nettools
   ];
 
-  patches = [ ./remove-hardcoded-locations.patch ];
+  patches = [
+    ./remove-hardcoded-locations.patch
+
+    # Use the exactly 32 byte long version of db_key_base with
+    # aes-256-gcm, see
+    # https://gitlab.com/gitlab-org/gitlab/-/merge_requests/53602
+    ./secrets_db_key_base_length.patch
```

Comment:
Can we use fetchpatch with https://gitlab.com/gitlab-org/gitlab/-/commit/dea620633d446ca0f53a75674454ff0dd4bd8f99.patch ?

Reply:
We could, but my concern is that it could break if I have to rebase or otherwise rewrite the history upstream.

Reply:
As long as you don't fetch the patch of your MR nothing will break.

Reply:
I meant that the commit could get garbage collected when nothing references it anymore, but maybe this isn't really a concern. I'll fix this and just update the url if I update the MR. :)

Reply:
At least github is not doing this. Not sure if girls eventually removes commits.